src/main/java/org/candlepin/controller/PoolManager.java

```diff
@@ -1242,11 +1244,16 @@ public List<PoolQuantity> getBestPools(Consumer consumer, Collection<String> pro
             activePoolDate = new Date();
         }
 
+        PoolQualifier qualifier = new PoolQualifier()
+            .addIds(fromPools)
+            .setOwnerId(ownerId)
+            .setConsumer(consumer)
+            .setActiveOn(activePoolDate);
+
         PoolFilterBuilder poolFilter = new PoolFilterBuilder();
         poolFilter.addIdFilters(fromPools);
```

Comment:
Forgot to remove PoolFilterBuilder from this method/class now that its replaced by PoolQualifier

Reply:
Additionally, while this PR removes all direct uses of the PoolFilterBuilder class, we still cannot remove it because it is extended by EntitlementFilterBuilder which is used for a lot of Entitlement-related queries. I wouldn't want to introduce an 'EntitlementQualifier' class, simply because Entitlements are going away and it would be a waste of time. But can we instead update EntitlementFilterBuilder to be what PoolFilterBuilder was, and delete the PoolFilterBuilder class?

Reply:
Removed PoolFilterBuilder from PoolManager and removed PoolFilterBuilder class. Then updated  EntitlementFilterBuilder. 

I agree on not refactoring the EntitlementFilterBuilder into a EntitlementQualifier class. Too much work for classes that will be going away!